packages/manager/modules/pci/src/components/hds/component.js

```diff
@@ -0,0 +1,15 @@
+import controller from './controller';
+import template from './template.html';
+
+const component = {
+  bindings: {
+    hds: '<',
+    model: '<',
```

Comment:
`model` is a one way binding property ? Does it stand for instance model ?

Reply:
yes @marie-j , it's model of hds component.